StripePayments/StripePayments/Source/API Bindings/Models/PaymentIntents/STPPaymentIntent.swift

```diff
@@ -147,6 +147,9 @@ public class STPPaymentIntent: NSObject {
     /// Country code of the merchant.
     @_spi(STP) public let merchantCountryCode: String?
 
+    /// Card brand choice settings for the merchant.
+    @_spi(STP) public let cardBrandChoice: STPCardBrandChoice?
```

Comment:
Another field in the v1/elements/sessions response that we're hacking into STPPaymentIntent :(

Do you plan to add it to STPElementsSession too?

Reply:
Also agree it's unfortunate, we should refactor this. In the future I'll add it to STPElementsSession for deferred and CustomerSheet, however for now it appears deferred is not supported with CBC until later this year.
https://stripe.slack.com/archives/C05GTTC4PB5/p1693420137876139?thread_ts=1693347390.543519&cid=C05GTTC4PB5